Fix contain-size-button-001+002.html

rwlbuis opened the pull request:
#### b850aebbf296f19378b75bd248c52377234dee9c
<pre>
Fix contain-size-button-001+002.html
<a href="https://bugs.webkit.org/show_bug.cgi?id=243300">https://bugs.webkit.org/show_bug.cgi?id=243300</a>

Reviewed by Tim Nguyen.

In WebKit buttons with auto width and height set intrinsic margins, causing these
tests to fail. Since margins do not affect size containment, set margins to zero
in order to exclusively test the size containment.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/contain-size-button-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/contain-size-button-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/contain-size-button-002-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/contain-size-button-002.html:

Canonical link: <a href="https://commits.webkit.org/252973@main">https://commits.webkit.org/252973@main</a>
</pre>

nt1m reviewed (approved):
Looks good, but please export to the WPT repo as well.

rwlbuis committed:
Fix contain-size-button-001+002.html
https://bugs.webkit.org/show_bug.cgi?id=243300

Reviewed by Tim Nguyen.

In WebKit buttons with auto width and height set intrinsic margins, causing these
tests to fail. Since margins do not affect size containment, set margins to zero
in order to exclusively test the size containment.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/contain-size-button-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/contain-size-button-002-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/contain-size-button-002-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/contain-size-button-002.html:

Canonical link: https://commits.webkit.org/252973@main
